R/mod_packageDependencies.R

```diff
@@ -97,6 +97,11 @@ packageDependenciesServer <- function(id, selected_pkg, user, changes, parent) {
       suggests(pkgref()$suggests[[1]] %>% dplyr::as_tibble())
     })
     
+    cards <- reactive({
+      req(pkgref())
+      build_dep_cards(data = dplyr::bind_rows(depends(), suggests()), loaded = loaded2_db()$name)
+    }) %>% shiny::bindCache(pkgref()) %>% shiny::bindEvent(pkgref())
```

Comment:
By binding the cache, the total number of dependencies doesn't update if you upload one during that same session.

Reply:
It's working now that I removed `bindCache()` thanks.